<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simple_dialog</class>
 <widget class="QDialog" name="simple_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Simple Reminder</string>
  </property>
  <property name="windowIcon">
   <iconset theme="remindor-qt">
    <normalon>:/icons/remindor-qt.svg</normalon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="reminder_label">
     <property name="text">
      <string>Remind me to...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="reminder_layout">
     <item>
      <widget class="QToolButton" name="reminder_error">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="error">
         <normalon>:/icons/delete.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reminder_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout">
     <item>
      <widget class="QPushButton" name="help_button">
       <property name="text">
        <string>Help</string>
       </property>
       <property name="icon">
        <iconset theme="help">
         <normalon>:/icons/help.png</normalon>
        </iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalspacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset theme="gtk-cancel">
         <normalon>:/icons/delete.png</normalon>
        </iconset>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset theme="add">
         <normalon>:/icons/add.png</normalon>
        </iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>reminder_error</sender>
   <signal>pressed()</signal>
   <receiver>reminder_edit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>11</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
